Makefile

```diff
@@ -201,20 +206,21 @@ rm-network:
 	@echo "\033\[32mremove network ${NETWORK}\033[0m"
 	docker network rm $(NETWORK)
 
-run-sgx-validation:
-	@echo "\033[32mrun docker sgx validation\033[0m"; \
-	docker run -d \
-	--net $(NETWORK) \
-	--restart=always \
-	--name $(prefix)sgx-validation \
-	-e SGX_MODE=$(SGX_MODE) \
-	-e NETWORK_ID=$(NETWORK_ID) \
-	-e RUST_LOG=$(RUST_LOG) \
-	--device $(sgx_device) \
-	-v  $(data_path)/enclave-storage:/crypto-chain/enclave-storage \
-	--workdir=/usr/local/bin \
-	$(IMAGE):$(TAG) \
-	bash ./run_tx_validation.sh
+# deprecated and replaced by abci
+# run-sgx-validation:
+# 	@echo "\033[32mrun docker sgx validation \033[0m"; \
+# 	docker run -d \
+# 	--net $(NETWORK) \
+# 	--restart=always \
+# 	--name $(prefix)sgx-validation \
+# 	-e SGX_MODE=$(SGX_MODE) \
+# 	-e NETWORK_ID=$(NETWORK_ID) \
+# 	-e RUST_LOG=$(RUST_LOG) \
+# 	--device $(sgx_device) \
+# 	-v $(data_path)/enclave-storage:/crypto-chain/enclave-storage \
+# 	--workdir=/usr/local/bin \
+# 	$(IMAGE):$(TAG) \
+# 	bash ./run_tx_validation.sh
```

Comment:
if this is reference, you can make markdown file for document and future reference

Reply:
I think we can just delete it and use Git to recover it in the future if needed.